src/niwidgets/niwidget_volume.py

```diff
@@ -192,9 +192,9 @@ def _plot_slices(
 
             # update the image
             imh.set_data(
-                np.flipud(np.rot90(data[slice_obj], k=1))
+                np.flipud(np.rot90(data[tuple(slice_obj)], k=1))
                 if views[ii] != "Sagittal"
-                else np.fliplr(np.flipud(np.rot90(data[slice_obj], k=1)))
+                else np.fliplr(np.flipud(np.rot90(data[tuple(slice_obj)], k=1)))
```

Comment:
This is currently breaking the formatting checker (see [here](https://travis-ci.org/nipy/niwidgets/jobs/643882767?utm_medium=notification&utm_source=github_status))

I think because this line is too long:

```suggestion
                else np.fliplr(
                    np.flipud(np.rot90(data[tuple(slice_obj)], k=1))
                )
```

Reply:
Sorry, did not notice the `black` formatter rules.